packages/plugin-essentials/sources/commands/_entry.ts

```diff
@@ -10,8 +27,9 @@ export default (clipanion: any) => clipanion
 
   .action(async ({cwd, version, args, stdout, ... env}: any) => {
     // berry --version
-    if (args.length === 1 && args[0] === `--version`) {
-      stdout.write(`v2.0.0\n`);
+    if (isVersion(args)) {
```

Comment:
I understand the rational for splitting it apart, but given how short it is I feel like in this situation it might be better to put the logic inline (like for `-h,--help`) so that we can easily see whether there are any overlaps. Does that make sense?

Reply:
` if (args.length === 1 && (args[0] === `--version` || args[0] === `-v`)) {`

^ Would seem more difficult to reason about and is quite long for a conditional, no?

Reply:
I find it ok. I feel like it makes it easier to read the command from the top to the bottom and know what it does, rather than having to jump to the helpers.

But it's not a blocker by any measure 🙂

Reply:
I reverted :slightly_smiling_face: